protocol-units/bridge/move-modules/sources/atomic_bridge.move

```diff
@@ -167,7 +170,82 @@ module atomic_bridge::atomic_bridge_counterparty {
     use aptos_framework::primary_fungible_store;
 
     #[test(aptos_framework = @0x1, creator = @atomic_bridge, moveth = @moveth, admin = @admin, client = @0xdca, master_minter = @master_minter)]
-    fun test_complete_transfer_assets(
+    fun test_complete_transfer_assets_non_minter(
+        client: &signer,
+        aptos_framework: &signer,
+        master_minter: &signer, 
+        creator: &signer,
+        moveth: &signer,
+    ) acquires BridgeTransferStore, BridgeConfig {
+        timestamp::set_time_has_started_for_testing(aptos_framework);
+        moveth::init_for_test(moveth);
+        let receiver_address = @0xcafe1;
+        let initiator = b"0x123"; //In real world this would be an ethereum address
+        let recipient = @0xface; 
+        let asset = moveth::metadata();
+
+        // the master minter sets client to be a minter
+        // moveth::add_minter(master_minter, signer::address_of(client));
+
+        //client now mints themselves 100 moveth
+        // moveth::mint(client, signer::address_of(client), 100);
+        // assert!(primary_fungible_store::balance(signer::address_of(client), asset) == 100, 0);
+
```

Comment:
Let's remove these commented out lines above from 187 - 192.

Reply:
Ah yep removed the unneeded comments from both tests.